components/molecules/Form/Form.tsx

```diff
@@ -66,21 +66,27 @@ const Form = ({ form }: { form: FormType }) => {
           <FormikForm>
             <FormContainer>
               {fields.map((field: FormFieldType, i: number) => {
-                const { type, name, label, placeholder, options, action, title } = field;
+                const { type, name, label, placeholder, options, title, accept, onClick, onChange, onBlur } = field;
                 const isInput = !(type === 'textarea' || type === 'select');
 
+                const actions = Object.entries({ onClick, onChange, onBlur }).reduce(
```

Comment:
Is it necessary to clear empty event handlers? What happens if you just bind them to `Field`?

Reply:
Yes, because in this case it will passed as `undefined` by default and formik field will not correctly work.
For example, I will not pass `onChange` - I type smth on the field - it's not typing to the field.